app/components/filmer/MovieDetails.tsx

```diff
@@ -0,0 +1,20 @@
+import { fetchMovie } from '../../lib/fetchMovies';
+
+export default async function Movie({ movieId }: { movieId: string }) {
+  const movie = await fetchMovie(movieId);
+
+  if (!movie) {
+    return <h2>Movie not found</h2>;
+  }
+  console.log('Fetched Movie:', movie?.title, movie?.year);
+  return (
+    <>
+      <h2>{movie?.title}</h2>
+      <img src={movie?.poster} width='200'></img>
+      <p>{movie?.fullplot.slice(0, 300) + '...'}</p>
```

Comment:
You could use "plot" here instead of "fullplot", then you dont need to use the slice at all.

Reply:
Yes, the length of the plot text will depend on how the styling turns out. This will do for now.